internal/server/placepage/golden/get_place_page_data_test.go

```diff
@@ -43,7 +43,7 @@ func TestGetPlacePageData(t *testing.T) {
 	ctx := context.Background()
 
 	for _, opt := range []*e2e.TestOption{
-		{},
+		// {},
```

Comment:
Should we keep this non-import-group test?

Reply:
Yea, it was a miss in previous PR